backend/app.ts

```diff
@@ -35,8 +36,8 @@ app.use(cors({
   origin: 'http://localhost:3000'
 }))
 
-app.use(express.json())
 app.use(passport.initialize())
+app.use(express.json())
```

Comment:
any reason the ordering was changed?

Reply:
Oh we did this when trying to debug I can put it back haha